bitmath/__init__.py

```diff
@@ -476,10 +476,10 @@ def best_prefix(self, system=None):
         if abs(self) < Byte(1):
             return Bit.from_other(self)
         else:
-            if not type(self) == Byte:
-                _inst = Byte.from_other(self)
-            else:
+            if type(self) is Byte:
```

Comment:
Not sure about this one. As I understood this check avoids unnecessary call to from_other() if self is Byte already. Unfortunatelly isinstance won't work this way because all other types are children of Byte.